src/zenml/services/terraform/terraform_service.py

```diff
@@ -356,6 +356,25 @@ def get_logs(
             "This method is not available for Terraform services."
         )
 
+    def get_outputs(self, output: Optional[str] = None) -> Dict[str, Any]:
```

Comment:
This probably needs to account for the fact that people will want to get multiple outputs at any one time. (As mentioned above). So it could accept an optional list of outputs that people want, and then output a dictionary containing whatever outputs were requested (or just all of them as the default)